arm-servicefabric/2016-09-01/swagger/servicefabric.json

```diff
@@ -0,0 +1,939 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "ServiceFabricClient",
+    "version": "2016-09-01"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
```

Comment:
Please add and triage following work item in your system.

Based on ARM team suggestion, we are recommending all RPs to add an operations API to their service. Here is an example how this API looks like https://github.com/Azure/azure-rest-api-specs/blob/ed4a7c5427117c2233f2aa54905d649c26e9a344/arm-billing/2017-02-27-preview/swagger/billing.json#L191